Linkurious/linkurious-rest-client
# renovate/configure

- Add renovate.json
- test adding node lts
- test constraints
- add labels
- w-e schedule
- Add security label
- set base branch
- list
- test base branch for security
- see if overide
- back to develop-stable
- limit concurrency
- move renovate file
- add file
